client/components/plans/plans-compare/index.jsx

```diff
@@ -31,6 +32,8 @@ import SidebarNavigation from 'my-sites/sidebar-navigation';
 import { SUBMITTING_WPCOM_REQUEST } from 'lib/store-transactions/step-types';
 import QueryPlans from 'components/data/query-plans';
 
+const hidePersonalPlan = plan => ! isEnabled( 'upgrades/personal-plan' ) && plan.product_slug === 'personal-bundle';
```

Comment:
This PR sounds like a good place to start using [these constants](https://github.com/Automattic/wp-calypso/blob/master/client/lib/plans/constants.js#L4-L6), at least for this new Personal plan. So much easier to refactor and less chances of a typo.